arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,837 @@
+        },
+        "counterValue": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the value of the statistic."
+        },
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the startTime of the statistic."
+        },
+        "endTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the endTime of the statistic."
+        },
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the id."
+        }
+      },
+      "description": "Definition of the statistic."
+    },
+    "StatisticsListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Statistics"
+          },
+          "description": "Gets or sets a list of statistics."
+        }
+      },
+      "description": "The response model for the list statistics operation."
+    },
+	"Usage": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the id of the resource."
+        },
+        "name": {
+          "$ref": "#/definitions/UsageCounterName",
+          "description": "Gets or sets the usage counter name."
+        },
+        "unit": {
+          "type": "string",
+          "description": "Gets or sets the usage unit name."
+        },
+        "currentValue": {
+          "type": "number",
+          "format": "double",
+          "description": "Gets or sets the current usage value."
+        },
+        "limit": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets max limit. -1 for unlimited"
+        }
+      },
+      "description": "Definition of Usage."
+    },
+	"UsageCounterName": {
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "Gets or sets the usage counter name."
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "Gets or sets the localized usage counter name."
+        }
+      },
+      "description": "Definition of usage counter name."
+    },
+    "UsageListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Usage"
+          },
+          "description": "Gets or sets usage."
+        }
+      },
+      "description": "The response model for the get usage operation."
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Gets subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
```

Comment:
Hi Vishrut,
 We have received exceptions for the below issues:
 [Azure Automation] validations (size, range, regexes) on properties and parameters missing  #1020
#1020
 [Azure Automation] Descriptions need to be corrected in all Azure Automation resources  #1021
#1021

Samer is aware of above issues. The two issues above will be revisited. These descriptions were inherited from Hyak specs and we won't be changing descriptions or validation regex at the moment